recipes/robotframework-ghareports/recipe.yaml

```diff
@@ -0,0 +1,55 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+context:
+  version: 0.8.0
+
+package:
+  name: robotframework-ghareports
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/r/robotframework-ghareports/robotframework_ghareports-${{ version }}.tar.gz
+  sha256: 4f083f7b60966ddd8dbabe3a2939ca8c49400ffd628e7214c11fa4370136c468
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation --disable-pip-version-check
+  python:
+    entry_points:
+      - ghareports = GHAReports.cli:main
+
+requirements:
+  host:
+    - pdm-backend
+    - pip
+    - python ${{ python_min }}.*
+  run:
+    - python >=${{ python_min }}
+    - robotframework >=4.0.0
+    - wrapt
+
+tests:
+  - python:
+      imports: GHAReports
+      pip_check: true
+      python_version:
+        - ${{ python_min }}.*
+        - 3.13.*
```

Comment:
Does this create two different import tests for the two versions? I haven't seen this before.

Reply:
yes! it's neat! Also need to fix SPDX :crying_cat_face:

Reply:
> fix SPDX

fixed that: anyhoo: yeah, this one little matrix bobbin really fills in a significant gap of CFEP-20 which pushes the burden of "trying new stuff," to end users.

Reply:
Indeed, to that point, the linter could suggest this for `noarch`... `${{ python_max }}` is a bad idea, basically demanding future work, but `${{ python_test_max }}` could be a quite useful convention for this specific case.

Reply:
> I haven't seen this before.

Cribbed from here: 

https://github.com/conda-forge/py-rattler-feedstock/blob/main/recipe/recipe.yaml#L54

You know it's kosher if the :wolf: writes it.